src/products/components/ProductStocks/ProductStocks.tsx

```diff
@@ -277,195 +266,192 @@ const ProductStocks: React.FC<ProductStocksProps> = ({
+                              paddingX={4}
+                              paddingY={5}
+                              borderRadius={4}
+                              onClick={() =>
+                                handleWarehouseStockAdd(warehouse.id)
+                              }
                             >
-                              <Paper className={classes.paper} elevation={8}>
-                                {warehousesToAssign.map(warehouse => (
-                                  <MenuItem
-                                    className={classes.menuItem}
-                                    onClick={() =>
-                                      handleWarehouseStockAdd(warehouse.id)
-                                    }
-                                  >
-                                    {warehouse.name}
-                                  </MenuItem>
-                                ))}
-                              </Paper>
-                            </Grow>
-                          )}
-                        </Popper>
-                      </div>
-                    </TableCell>
-                  </TableRowLink>
-                </ClickAwayListener>
+                              <Text>{warehouse.name}</Text>
+                            </List.Item>
+                          </Dropdown.Item>
+                        ))}
+                      </List>
+                    </Box>
+                  </Dropdown.Content>
+                </Dropdown>
               )}
             </TableBody>
           </Table>
         )}
       {data.isPreorder && (
-        <CardContent>
-          <Typography variant="caption" className={classes.caption}>
-            <FormattedMessage {...messages.preorderEndDateSetup} />
-          </Typography>
-
-          {data.hasPreorderEndDate && (
-            <div className={classes.dateTimeInputs}>
-              <DateTimeTimezoneField
-                name={"preorderEndDateTime"}
+        <DashboardCard.Content>
+          <Box display="grid" gap={5}>
+            <Text variant="caption">
+              <FormattedMessage {...messages.preorderEndDateSetup} />
+            </Text>
+            {data.hasPreorderEndDate && (
+              <Box>
+                <DateTimeTimezoneField
+                  name={"preorderEndDateTime"}
+                  disabled={disabled}
+                  futureDatesOnly
+                  fullWidth={false}
+                  error={localFormErrors.preorderEndDateTime}
+                  value={data?.preorderEndDateTime}
+                  onChange={event =>
+                    onChangePreorderEndDate({
+                      target: {
+                        name: "preorderEndDateTime",
+                        value: event,
+                      },
+                    })
+                  }
+                />
+              </Box>
+            )}
+            {/* @ts-ignore */}
+            <Box __alignSelf="end">
```

Comment:
**suggestion**: lets add alignSelf to sprinkles

Reply:
This is my plan - I have a couple of other fixes as well so I will push them in bulk PR